### PR TITLE
fix/1712: Update activity filter behavior

### PR DESCRIPTION
**Pull request by zamuzakki**
This is PR for #1712
- [x] By default, all Activity will be deselected.
- [x] By default, reports will show all records, be it having activity or not.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 32 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 34 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 29 sec

File: django_project/frontend/utils/data_table.py [hunk 1]
```diff
@@ -180,21 +180,19 @@ def get_report_filter(request, report_type):
...
-    elif report_type == ACTIVITY_REPORT:
-        filters[
-            default_activity_field
-        ] = ActivityType.objects.values_list('id', flat=True)
+    # else:
+    #     filters['id__in'] = AnnualPopulation.objects.exclude(
+    #         id__in=AnnualPopulationPerActivity.objects.values_list(
+    #             'annual_population', flat=True
+    #         ).distinct()
+    #     )
```
Comment (dimasciput) [inline django_project/frontend/utils/data_table.py:194]:
```suggestion
```

**Review by dimasciput — APPROVED** [review]

**Comment by dimasciput** [comment]
@zamuzakki please fix the conflicts

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 26 sec

**Comment by zamuzakki** [comment]
@dimasciput I've resolved the conflict

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 57 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 41 sec

**Pull request merged**